### PR TITLE
feat: handling constants documentation

### PR DESCRIPTION
**Pull request by Pratik-commutatus**

### CODE REVIEW

**Review by AdityaTiwari2102 — APPROVED** [review]

File: domains/engineering/well-documented/handling-constants.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  - Example export: 
+    ```javascript
+    export const FOO = 'bar';
+    ```
+  - Example usage: 
+    ```javascript
+    import { FOO } from '../constants';
+    ```
+    
+2. **SCSS constants**
```
Comment (gpalsingh) [inline domains/engineering/well-documented/handling-constants.md:94]: - These instructions are not limited to just Angular. React and Vue can use the same pattern
- I'm assuming this pattern can be used in ROR too

Based on this, I suggest we move this info to its own section

Also, in the ROR example, we use [partials](https://sass-lang.com/guide) which we can also use here. i.e. `_variables.scss`, `_helpers.scss`, etc. for the files that are imported into other `scss` files only

Comment (Pratik-commutatus) [reply domains/engineering/well-documented/handling-constants.md:94]: @gpalsingh - Good suggestion. 
Although one point to note would be: Although the basic logic regarding the functionality is same, one thing that would be different for every framework would be the path in which the partial should be stored. For example, in Rails, it is recommended to be stored in a default 'base' folder. Similarly, Guru has provided an example with the following path for NextJS: `@styles/design-system.scss`
To include all this information in the separate section that you suggested, we would again need to section it out based on the different frameworks, which I'm not sure is better than what we have currently.
What do you think?

Comment (gpalsingh) [reply domains/engineering/well-documented/handling-constants.md:94]: For the NextJS example, I believe that it is not limited to NextJS. `@styles` is the name of the directory. (Correct me if I'm wrong here @Gurubalan-GIT )

From what I see there are different conventions being used by different developers which are not limited to any framework.

This is a good chance to compare all the different approaches and come up with a single most efficient solution
@Pratik-commutatus

**Review by Gurubalan-GIT — APPROVED** [review]

File: domains/engineering/well-documented/handling-constants.md (hunk 1 above)
Comment (Gurubalan-GIT) [reply domains/engineering/well-documented/handling-constants.md:94]: @gpalsingh -> Second that, `@styles` is just the directory and the name is such due to `jsconfig.json` file being there.

Comment (Pratik-commutatus) [reply domains/engineering/well-documented/handling-constants.md]: @gpalsingh - I have split the SCSS constants into a separate section as per your suggestion. For the issue regarding having multiple framework-specific path examples, I have added a general best practice for the path in which we need to save the partials. I believe this structure can be followed across all the framewroks.

Comment (gpalsingh) [reply domains/engineering/well-documented/handling-constants.md]: Looks good bro 👍🏽

**Pull request merged**